### PR TITLE
Feral Mechanic

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
Content "add feral mechanics"

#### Purpose of change
adding the feral workers, as part to diverse the worker category of monsters in game (right now there is only technicians for such purposes)

#### Describe the solution
added feral worker, and added it to all monster groups where zombie technicians met

#### Describe alternatives you've considered
not adding them
adding some another workers
convert this one to zeds
#### Testing

works fine, spawns correct

#### Additional context
maybe need some polishing, especially in description, i think

### CODE REVIEW

**Comment by Alex-Folts** [comment]
You should consider giving him tire iron, as a weapon or just as one of drop items.

**Comment by GuardianDll** [comment]
For what? There is another feral with such tool

**Comment by a-chancey** [comment]
Nice! These will go well with the zombie mechanics I’ve started on and never finished with a new location I’m planning.

File: data/json/monsters/feral_humans.json [hunk 1]
```diff
@@ -163,6 +163,16 @@
     "death_drops": "feral_humans_death_drops_axe",
     "fungalize_into": "mon_feral_human_axe_fungal_infected"
   },
+  {
+    "id": "mon_feral_human_tool",
+    "type": "MONSTER",
+    "name": "feral mechanic",
+    "description": "This wild human-like creature was a liable mechanic before the cataclysm.  You can say it by his worker uniform, or by his blood-covered hammer in hand, or by his all-hating view.",
```
Comment (Maleclypse) [inline data/json/monsters/feral_humans.json:170]:
```suggestion
    "description": "This wild human-like creature was probably mechanic before the cataclysm.  You can see it by his worker uniform, or by his blood-covered hammer in hand, or by his all-hating view.",
```

**Pull request merged**